pkgs/by-name/ko/koboldcpp/package.nix

```
@@ -18,6 +19,9 @@
   openblas,
 
   cublasSupport ? config.cudaSupport,
+  # You can find a full list here: https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/
+  # For example if you're on an GTX 1080 that means you're using "Pascal" and you need to pass "sm_60"
+  arches ? cudaPackages.cudaFlags.arches or [ ],
```

```suggestion
  cudaArches ? cudaPackages.flags.arches,
```

- A more specific name might be preferable if 1) one was to add rocm support, 2) one were to add a package named "arches" (I hope this never happens, that's a bad name)
- The `or [ ]` is unnecessary, at this point we'd be afraid to change the interface of `cudaPackages.flags`

But IMO this is already good to go